activity/s3/activity.go

```diff
@@ -45,6 +50,7 @@ func New(ctx activity.InitContext) (activity.Activity, error) {
 		}
 		act.awsSession = session.Must(session.NewSession(&aws.Config{
 			Region: aws.String(region),
+			Credentials: credentials.NewEnvCredentials(),
```

Comment:
Not sure this is needed, the default behavior of the SDK should sort out the credentials lookup, could be environment, shared credential file or IAM role.

Reply:
I got authentication issues if this line was not added.